### PR TITLE
Make messages in 'Note To Self' chat room take full available screen width

### PR DESCRIPTION
**Pull request by arkascha**
### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
screenshot_befe | screenshot_after

### DESCRIPTION

This PR implements the change requested in [issue #3920](https://github.com/nextcloud/talk-android/issues/3920). 

Messages bubbles in "NoteToSelf" chat room use the full available width now. There is no sense in leaving a start margin, since no other participant can add messages to that chat room. 

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included or not needed
- [x] 🔖 Capability is checked or not needed 
- [x] 🔙 Backport requests are created or not needed: `/backport to stable-xx.x`
- [x] 📅 Milestone is set
- [x] 🌸 PR title is meaningful (if it should be in the changelog: is it meaningful to users?)

### CODE REVIEW

**Comment by arkascha** [comment]
I fail to understand what the failed analysis is actually trying to say. 
I do not see any detail reported by any of the checks that somehow corresponds to my changes. 
Is there anything I miss? Anything I need to do, to fix?

**Comment by mahibi** [comment]
> I fail to understand what the failed analysis is actually trying to say. I do not see any detail reported by any of the checks that somehow corresponds to my changes. Is there anything I miss? Anything I need to do, to fix?

don't worry about the failed analysis.
Thank you for contributing!  I will try it out tomorrow, code wise it looks good :+1:

**Review by mahibi — APPROVED** [review]

**Comment by mahibi** [comment]
works fine :1st_place_medal: 
i enabled auto merge. could you rebase on newest master @arkascha ? 

If you like to continue contributing, just let me know if you are interested to get access to talk android repo so you could create branches in it...

**Review by rapterjet2004 — APPROVED** [review]

**Comment by arkascha** [comment]
@mahibi I rebased and fixed an issue pointed out by Codacy. 
I did NOT squasch the two commits, I intentionally left the in the PR since they target different issues. I personally prefer to have such things separate in the git history in my own projects. 
Auto merging does not seem to continue though, "This workflow requires approval from a maintainer. " is mentioned above.

**Comment by arkascha** [comment]
@mahibi About access to your repository ... Thanks for the trust in me, but for now I am happy with working on my own clone, I think. As long as PRs like this one are OK for you. This PR was just a test balloon for some bigger idea I would like to try. No idea if I succeed, so I do not want to create any hassle.

**Comment by mahibi** [comment]
> @mahibi About access to your repository ... Thanks for the trust in me, but for now I am happy with working on my own clone, I think. As long as PRs like this one are OK for you. This PR was just a test balloon for some bigger idea I would like to try. No idea if I succeed, so I do not want to create any hassle.

Sure, working with the fork is okay for now.
Please feel free to share the bigger idea you have in mind (best by creating an issue). Just want to avoid that work is done and soon afterwards it's obsolete because some technology is replaced etc. 
E.g. the ViewHolders will be affected/replaced when switching to Jetpack Compose etc..

**Comment by arkascha** [comment]
@mahibi There seems to be an issue again. 2 failed checks (which I fail to make much sense of) and one check still expected (which seems odd).

**Comment by AndyScherzinger** [comment]
@arkascha I bumped the detekt score by 1 (this PR add another detekt warning / issue) yet it is hard spot spot which one is the new one since the report simply lists all finding. Not too much of an issue, hence I bumped the score, so detekt should now turn green.

**Comment by arkascha** [comment]
@AndyScherzinger , @mahibi Thanks, so now we are down to just one failed check. To me that one sounds like a broken setup of the tool chain. Anything I can do in this?

**Comment by AndyScherzinger** [comment]
A bit of both @arkascha the check failed to upload the result but also fails compared to master, see https://github.com/nextcloud/talk-android/actions/runs/11724053977/job/32657114027?pr=4408#step:6:100

I pushed a commit fixing some spotbugs issue to see/get to a score not higher in terms of number of warnings compared to master. If we are back at 108 warning (let's see what the check returns) than I can force merge 👍

**Review by AndyScherzinger — APPROVED** [review]

**Comment by AndyScherzinger** [comment]
So.... Spotbugs is back at the same amount of warnings we have on master, hence everything is good to go.

Thanks a lot for this contribution @arkascha - highly appreciated and thank you for your patience in getitng everythign in order to get it to be merged and sorry for the (still) broken analysis Github action.

**Comment by arkascha** [comment]
Thanks for helping out @AndyScherzinger!
Looking through the last commit you added I have the impression that none of those things you fixed are related to my changes. So I still wonder what failing checks I added with my changes...

Anyway, great that things are merged now. It has been a pleasure to contributed a tiny detail. 
And thanks for the awesome app at all!

**Comment by arkascha** [comment]
I assume the related issue can be closed now. 
I do not have the required permission to do that, though.

**Comment by AndyScherzinger** [comment]
Yes, thanks for the ping - closed the issue 👍

**Pull request merged**
